server/settings/main.py

```diff
@@ -47,10 +47,10 @@ class PostLaunchHookMapping(BaseSettingsModel):
 class PostLaunchHookSettings(BaseSettingsModel):
     """Change task status on application launch.
 
-    Changeo of status is based on mapping. Each item in mapping define new
-    status which is used based on current status/es. Special value for current
-    statuses is '__any__', in that case the new status is always used. And if
-    new status name is '__ignore__', the change of status is skipped if current
+    Change of status is based on mapping. Each item in mapping defines new
+    status which is used based on current status(es). Special value for current
+    statuses is `__any__`, in that case the new status is always used. And if
```

Comment:
I'm using `'` in docstrings for easier readibility, especially if the valus is a string.

Reply:
The issue is that on the site it converts it to markdown, and thus makes `__ignore__` the word '__ignore__' which is not the value the user should be seeing.

Reply:
> Also, this should technically bump the server addon version - but at the same, do we really care if this particular doc change doesn't show yet for the user if it weren't updated.

Addons in repositories have standard development workflow. PRs are merged to `develop` and bump version when is merged to `main`.

> on the site

Which site?

Reply:
I'm blind, it's change in settings model, yeah make sense.

> Side question: What do you call 'this site'? Ayon server? Ayon server frontend?

In this case I would use Settings UI. But can be AYON Web, Web UI.